lib/Server.js

```diff
@@ -115,7 +115,14 @@ class Server {
 
     this.sockets = [];
 
-    this.watchOptions = options.watchOptions || {};
+    if (!options.watchOptions) {
+      options.watchOptions = {};
+    }
+    // ignoring node_modules folder by default
+    options.watchOptions.ignored =
+      options.watchOptions.ignored || /node_modules/;
```

Comment:
I think we need use here `[/node_modules/]`. also need add tests to avoid future regressions